andela/ah-infinity-stones-frontend
# #161966867 Setup react router version 4

#### What does this PR do?
- Sets up react router for our client

#### Description of Task to be completed?
> npm install create-router-dom
> create a views folder to hold UI components
> Create view components ( Home, login & Navbar )
> Create appropriate routes in app.jsx

#### How should this be manually tested?
`$ git clone -b ch-setup-react-router-version4-161966867 https://github.com/andela/ah-infinity-stones-frontend`
`$ cd ah-infinity-stones-frontend`
> Install node modules specified in `package.json`
`$ npm install`
`$ npm start`

> click the links in the nav bar and notice the page never refreshes since the router has taken charge!

#### What are the relevant pivotal tracker stories?
[#161966867](https://www.pivotaltracker.com/story/show/161966867)


#### Screenshots
<img width="1439" alt="screenshot 2019-01-10 at 20 28 21" src="https://user-images.githubusercontent.com/16904946/50985885-6f2a0780-1516-11e9-80dd-2481464889f9.png">
<img width="1440" alt="screenshot 2019-01-10 at 20 28 30" src="https://user-images.githubusercontent.com/16904946/50985871-689b9000-1516-11e9-9320-f51d0b445fb0.png">
<img width="1439" alt="screenshot 2019-01-10 at 20 28 38" src="https://user-images.githubusercontent.com/16904946/50985855-60435500-1516-11e9-91c8-903bd8eefeb7.png">
